application/src/main/resources/locale/blueprint/horizons/type.description.csv

```diff
@@ -161,6 +161,6 @@ blueprint.horizons.type.description.modified_shard_cannon_fixed_small,"Pre-Engin
 blueprint.horizons.type.description.e2_sirius_corporation_ax_missile_rack,A pre-engineered Size 2 AX Missile Rack with increased ammo capacity and fire rate with reduced reload time at the cost of slightly reduced damage and increased mass. The warheads are tipped with a compound hazardous for Thargoid vessels. The module must be unlocked repeatedly in order to obtain multiple copies.,,,,,,"Блок ракет АИ 2 Класс в Заводской комплектации с увеличенным боезапасом, скорострельностью и с уменьшенным временем перезарядки за счет небольшого уменьшения урона и увеличения массы. Боеголовки начинены составом, опасным для таргоидских кораблей. Модуль необходимо многократно разблокировать, чтобы получить несколько копий."
 blueprint.horizons.type.description.c3_sirius_corporation_ax_missile_rack,A pre-engineered Size 3 AX Missile Rack with increased ammo capacity and fire rate with reduced reload time at the cost of slightly reduced damage and increased mass. The warheads are tipped with a compound hazardous for Thargoid vessels. The module must be unlocked repeatedly in order to obtain multiple copies.,,,,,,"Блок ракет АИ 3 Класс в Заводской комплектации с увеличенным боезапасом, скорострельностью и с уменьшенным временем перезарядки за счет небольшого уменьшения урона и увеличения массы. Боеголовки начинены составом, опасным для таргоидских кораблей. Модуль необходимо многократно разблокировать, чтобы получить несколько копий."
-blueprint.horizons.type.description.engineer,,,,,,,
-blueprint.horizons.type.description.synthesis,,,,,,,
-blueprint.horizons.type.description.caustic_sink_launcher,Module that utilises a form of cathodic protection to temporarily protect a ship's hull from corrosion. Corroded material is diverted to an insulated container which can be ejected.,,,,,,
\ No newline at end of file
+blueprint.horizons.type.description.engineer,,,,,,,Инженер
+blueprint.horizons.type.description.synthesis,,,,,,,Синтез
```

Comment:
Above 2 values are intentionally empty, can you revert them?

Reply:
Done. Sorry for the inconvenience.